packet.go

```diff
@@ -662,7 +662,8 @@ type sshFxpOpenPacket struct {
 	ID     uint32
 	Path   string
 	Pflags uint32
-	Flags  uint32 // ignored
+	Flags  uint32
```

Comment:
My primary concern here is that this `Flags` does not support all of the `SSH_ATTR` possibilities, but there is nothing to warn that it _only_ supports `SSH_FILEXFER_ATTR_PERMISSIONS`.

It’s also still quite a bit of a hack, and every hack we make can limit our future. Consider: we’ve already added an `OpenFile` and its signature doesn’t match `os.OpenFile`; we’re stuck with all of the errors spelled `ErrSshFxOk` even though they’ve all already been replaced by `ErrSSHFxOk`; By adding the `ReadFromWithConcurrency` we’re kind of stuck with exposing it, even though our new sequential issuing eliminates almost all of the issues that were theoretically possible with enabling concurrency; we’re on the precipice of handling client side packets with the new `filexfer` package that handle all features of all packets, and avoids lots of the “hacky” packet things here.

I’d really rather people just make due without a “correct” `OpenFile` until we have a chance to cut `v2` where we can fix the signature of `OpenFile` itself.

Reply:
Fair enough. Feel free to close.